wrapper/tests/test_pre_copy.py

```diff
@@ -7,11 +7,12 @@ class TestPreCopy(unittest.TestCase):
 
     basic_data = {
         'output_format': 'raw',
-        'vmware_uri': 'vpx://example.com',
+        'vmware_uri': 'esx://esx1.example.com',
```

Comment:
You are removing code coverage by testing only the esx transport even though the code supports more than that.

Reply:
Ok. Then I've added test for SSH.